crates/atlaspack/src/requests/path_request.rs

```
@@ -66,11 +66,21 @@ impl Request for PathRequest {
     let mut invalidations = Vec::new();
 
     for resolver in request_context.plugins().resolvers()?.iter() {
-      let resolved = resolver.resolve(ResolveContext {
+      let result = resolver.resolve(ResolveContext {
         dependency: Arc::clone(&self.dependency),
         pipeline: parsed_pipeline.clone(),
         specifier: String::from(specifier),
-      })?;
+      });
+
+      let resolved = match result {
+        Ok(result) => result,
+        Err(error) => {
```

hm.... that's not good, why do we need to eat these errors?

---

We need to update our diagnostic type to allow multiple errors/sub-errors.